frontend/src/components/PreviewReviewModal.tsx

```diff
@@ -0,0 +1,141 @@
+import {
+  Box,
+  HStack,
+  Image,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  Stack,
+  Tag,
+  Text,
+} from "@chakra-ui/react";
+import React from "react";
+
+/**
+ * The model defining the props for the Preview Review Modal component
+ */
+interface PreviewReviewModalProps {
```

Comment:
I decided not to use `ReviewResponseDTO` because the component's props should not be coupled with the API response

Reply:
Seems a bit tedious to have to pass each of these props individually in the parent element, would it make more sense to accept a single review object as a prop

Reply:
I decided against accepting a review object as a prop because that tightly couples this component to the API response of the reviews resource. The props of the component should be independent to the response of the API. The other alternative would be to accept `PreviewReviewModalProps` as a prop but it seems like eslint prefers the parameters be restructured.

Reply:
Ideally, we'd have a generic "Review" type that we could pass rather than a DTO, but not sure what the best practice is